docs/deploy-full-rp-service-in-dev.md

```diff
@@ -338,7 +338,7 @@
     export RESOURCEGROUP=myResourceGroup
     ```
 
-1. Create the resource group if it doesn't exist
+1. Create the  if it doesn't exist
```

Comment:
NIT: I am not sure this deletion was intended. If so the sentence has become unclear.